products/protractor.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
```

Comment:
```suggestion
tags: google javascript-runtime
```

This tag is new, let's add it later if there are more than one product that uses it.

Reply:
Got it, done!